.github/workflows/docker-publish.yml

```diff
@@ -73,6 +73,40 @@ jobs:
             # keep major version, e.g., v1, tag updated to latest release
             type=semver,pattern=v{{major}},enable=${{startsWith(github.ref, 'refs/tags/v')}}
 
+      - name: Find latest vcpkg version
```

Comment:
I tried for a while to look for a way to get Dependabot to do this for us, but came up empty. I still think the Dependabot model where your get a dedicated PR for your dependency updates is the way to go.

Could we do these that you've added here in a new workflow that triggers on a schedule and uses something like [create-pr-action](https://github.com/technote-space/create-pr-action) to put up a PR with just the .vcpkg_version file updated?

Reply:
I can add a scheduled workflow that calls this one and creates the PR if there's a new release.